release-notes-2.1.0.md

```diff
@@ -4,5 +4,6 @@
 ${version-number}
 
 #### New Features
+- **SCMOD-8516**: Extend the TLS/SSL cipher hardening of the Java base images to disallow TLSv1.1
```

Comment:
This is a breaking change so it will need to be listed as such in the release notes and the version number should be updated appropriately.